packages/cli/src/controllers/passwordReset.controller.ts

```diff
@@ -231,7 +241,7 @@ export class PasswordResetController {
 				resetPasswordToken,
 				resetPasswordTokenExpiration: MoreThanOrEqual(currentTimestamp),
 			},
-			relations: ['authIdentities'],
+			relations: ['authIdentities', 'globalRole'],
```

Comment:
This change seems to imply we'll need this in order to check the role in a users quota guard but there's no such check below? This endpoint doesn't need guarding? Or why is this addition needed?

Reply:
I would have swore that without this in:
```
@AfterLoad()
	computeIsOwner(): void {
		this.isOwner = this.globalRole?.name === 'owner';
	}
```
The `globalRole` would be undefined, but that doesn't seem to be the case. So removed. Thanks!